flaml/tune/searcher/blendsearch.py

```diff
@@ -693,6 +700,8 @@ def on_trial_result(self, trial_id: str, result: Dict):
     def suggest(self, trial_id: str) -> Optional[Dict]:
         """choose thread, suggest a valid config."""
         if self._init_used and not self._points_to_evaluate:
+            if self._cost_budget and self._cost_used >= self._cost_budget:
+                return None

```

Comment:
```suggestion
                return 
```

Reply:
Why remove "None" from "return None"? `suggest` should return `config` if it ends sucessfully, I think `return None` here indicates that this function should return some value, just at this point, it returns None.

Considering `def suggest(self, trial_id: str) -> Optional[Dict]:`, maybe we can use `return {}`?